src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -296,7 +296,7 @@ const ChatInfoScreen = () => {
 
   return (
     <SafeAreaView style={styles.container}>
-      <StatusBar translucent={false} />
+      <StatusBar translucent={false} barStyle={'light-content'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `barStyle` prop of the `StatusBar` component has been set to `'light-content'`. This changes the color of the status bar content to light. Make sure this is consistent across all screens for a uniform user experience and that it provides sufficient contrast against the background color for readability.
```diff
-      <StatusBar translucent={false} />
+      <StatusBar translucent={false} barStyle={'light-content'} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->